ext/ddtrace.c

```diff
@@ -2048,7 +2050,7 @@ static PHP_FUNCTION(set_priority_sampling) {
         RETURN_FALSE;
     }
 
-    if (global || !DDTRACE_G(active_stack) || !DDTRACE_G(active_stack)->root_span) {
+    if (global || DDTRACE_G(disable) || !DDTRACE_G(active_stack)->root_span) {
```

Comment:
This assumes if not disabled that there's an active stack, but in the rshutdown handler there is this:

    DDTRACE_G(active_stack) = NULL;

Which means there is a possibility that it's null, even though it's remote. Thoughts?

Reply:
Yeah, agreed, see my new commit :-)